v22.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -311,6 +315,82 @@ For CockroachDB clusters running in other environments, `implicit` authenticatio
+    ~~~sql
+    BACKUP DATABASE <database> INTO 'gs://{bucket name}/{path}?AUTH=implicit&ASSUME_ROLE={service account name}iam.gserviceaccount.com';
+    ~~~
+
+    CockroachDB also supports authentication for assuming roles when taking encrypted backups. To use with an encrypted backup, pass the `ASSUME_ROLE` parameter to the KMS URI as well as the bucket's: 
+
+    ~~~sql
+    BACKUP DATABASE <database> INTO 'gs://{bucket name}/{path}?AUTH=implicit&ASSUME_ROLE={service account name}iam.gserviceaccount.com'
+    WITH kms = 'gs:///projects/{project name}/locations/us-east1/keyRings/{key ring name}/cryptoKeys/{key name}?AUTH=IMPLICIT&ASSUME_ROLE={service account name}iam.gserviceaccount.com';
+    ~~~
+
+    For more information on Google Cloud Storage KMS URI formats, see [Take and Restore Encrypted Backups](take-and-restore-encrypted-backups.html).
+
+    {{site.data.alerts.callout_info}}
+    CockroachDB supports assume role authentication for changefeeds emitting to Google Cloud Pub/Sub sinks. The process to set up assume role for Pub/Sub works in a similar way, except that you will provide the final service account with the "Pub/Sub Editor" role at the project level. See the [Changefeed Sinks](changefeed-sinks.html#google-cloud-pub-sub) page for more detail on the Pub/Sub sink.
+    {{site.data.alerts.end}}
+
+#### Role Chaining
+
+Beyond a user assuming a role, it is also possible to "chain" roles to create a path for users to assume roles to particular operations. Role chaining allows a user to assume a role through an intermediate role(s) instead of the user directly assuming a role. In this way, the role chain passes the request for access to the final role in the chain. Role chaining could be useful when a third-party organization needs access to your Google Cloud Storage bucket to complete a bulk operation. Or, your organization could grant roles based on limited-privilege levels.
+
+Following from the previous setup section, if you want to add an intermediate account to the chain of roles, it is necessary to ensure each service account has granted the "Service Account Token Creator" role to the previous account in the chain. See [step 3](#service-account-token-granting) in the previous section to add this role on a service account.
+
+In a chain of three roles, A, B, C:
+
+Service Account A | &larr; Service Account B (intermediate accounts) | &larr; Service Account C (final account)
+-----------------+----------------+---------
+Credentials included in `AUTH=implicit` or `specified` | Grants access to **A** with the Service Account Token Creator role | Grants access to **B** with the Service Account Token Creator role<br>Access to the resource e.g., storage bucket
+
+- The intermediate account will delegate the request to account C.
+- The final service account will request the credentials that account A requires.
+
+When passing a chained role into `BACKUP`, it will follow this pattern with each chained role separated by a `,`: 
```

Comment:
nit: just because we are indicating the `,` character immediately before another bit of punctuation, `:`, a single word in between makes this much more readable:

> ... separated by a `,` character: